### ISSUE TITLE
add default path , so I can use this more easy

### ISSUE THREAD

**Issue by zew13** [jlindenbaum/res-resizer#3]
```
if args.folderPath == None:

    from os.path import join, dirname, abspath, exists
    folderPath = join(dirname(abspath(__file__)),"res/drawable-xhdpi")
    if exists(folderPath):
        args.folderPath = folderPath 
```

**Comment by jlindenbaum** [comment]
I'd be glad to add this in, if you could make this a pull request it would be a lot easier to merge.

**Comment by jlindenbaum** [comment]
This has been addressed with #5. Please use the new --prod option.
